### PR TITLE
wording tweaks

### PR DESCRIPTION
**Pull request by tkimnguyen**

### CODE REVIEW

File: docs/about.rst [hunk 1]
```diff
@@ -102,12 +102,12 @@ Prefer keyword arguments
...
-Example code in `plone.api` will use this style, and we recommend users to follow this convention.
-For the curious, here are the reasons:
+Example code in `plone.api` will use this style, and we recommend users follow this convention.
+For the curious, here are the reasons why:
 
-#. There will never be a doubt when writing a method on whether an argument should be positional or not.
+#. There will never be any doubt when writing a method whether an argument should be positional or not.
    Decision already made.
-#. There will never be a doubt when using the API on which argument comes first, or which ones are named/positional.
+#. There will never be asy doubt when using the API on which argument comes first, or which ones are named/positional.
```
Comment (adamcheasley) [inline docs/about.rst]: Typo here, "asy" should be "any"

Comment (tkimnguyen) [inline docs/about.rst]: Done - thx for catching that!

**Comment by gforcada** [comment]
@tkimnguyen would you mind rebasing so it can be merged?

**Comment by tkimnguyen** [comment]
@gforcada er sorry, beyond my depth a bit here... rebase to what?

**Comment by tkimnguyen** [comment]
I tried following the GitHub instructions here but it seemed to have no effect.

```
git checkout -b tkimnguyen-master master
git pull git://github.com/tkimnguyen/plone.api.git master

Step 2: Merge the changes and update on GitHub.

git checkout master
git merge --no-ff tkimnguyen-master
git push origin master
```

```
bash-3.2$ git status
On branch master
Your branch is up-to-date with 'origin/master'.
nothing to commit, working tree clean
bash-3.2$ git remote -v
origin	git@github.com:tkimnguyen/plone.api.git (fetch)
origin	git@github.com:tkimnguyen/plone.api.git (push)
bash-3.2$ git checkout -b tkimnguyen-master master

fatal: A branch named 'tkimnguyen-master' already exists.
bash-3.2$ bash-3.2$ git checkout tkimnguyen-master 
Switched to branch 'tkimnguyen-master'
bash-3.2$ git pull git://github.com/tkimnguyen/plone.api.git master
From git://github.com/tkimnguyen/plone.api
 * branch            master     -> FETCH_HEAD
Already up-to-date.
bash-3.2$ git checkout master
Switched to branch 'master'
Your branch is up-to-date with 'origin/master'.
bash-3.2$ git merge --no-ff tkimnguyen-master
Already up-to-date.
bash-3.2$ git push origin master
Everything up-to-date
bash-3.2$ 
```

**Comment by gforcada** [comment]
@tkimnguyen forking rather than creating the branches in plone org itself makes it harder to help you on the rebase.

To shortcircuit it although I did the rebase myself and push a new branch in plone org: so I'm closing this and we can follow on https://github.com/plone/plone.api/pull/336

**Comment by tkimnguyen** [comment]
thx @gforcada - I have been reluctant to mess with the main repo for fear of !@#$@ing something up but ok :)

**Pull request closed without merging**